lms/djangoapps/course_wiki/plugins/markdownedx/__init__.py

```diff
@@ -1,2 +1,3 @@
 # Make sure wiki_plugin.py gets run.
-from course_wiki.plugins.markdownedx.wiki_plugin import ExtendMarkdownPlugin
+#TODO (flying_pi) commented for run lms
```

Comment:
shouldn't the import be left as is?

Reply:
Thank you, nice catch. It is for my local deployment, I remove it